client/src/app/components/nova/block/BlockTangleState.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from "classnames";
+import moment from "moment";
+import React, { useEffect, useState } from "react";
+import Tooltip from "../../Tooltip";
+import { BlockState, u64 } from "@iota/sdk-wasm-nova/web";
+import { BlockFailureReason } from "@iota/sdk-wasm-nova/web/lib/types/models/block-failure-reason";
+import "./BlockTangleState.scss";
+
+export interface BlockTangleStateProps {
+    /**
+     * The Block status.
+     */
+    status: BlockState;
+
+    /**
+     * The issuing time.
+     */
+    issuingTime: u64;
+
+    /**
+     * The failure reason.
+     */
+    failureReason?: BlockFailureReason;
+}
+
+const BlockTangleState: React.FC<BlockTangleStateProps> = ({ status, issuingTime, failureReason }) => {
+    const [ago, setAgo] = useState<string | undefined>();
+
+    useEffect(() => {
+        setAgo(moment(Number(issuingTime) / 1000000).fromNow());
```

Comment:
We changed this on stardust recently with [this](https://github.com/iotaledger/explorer/pull/963). I think we should do the same on nova